### DISCUSSION TITLE
Virtual touchscreen is not active

### DISCUSSION THREAD

**Discussion by BroRidersMTL** [tesla-android/issue-tracker#41] [Q&A]
Hello, im getting this message "virtual touchscreen is not active. Wait for android to boot up."

Message goes away but nothing clicks using mouse not transferred to car yet

**Comment by BroRidersMTL** [comment]
Resolved

**Reply by ochompsky** [reply]
how?

**Reply by BroRidersMTL** [reply]
U need to save the settings for the DHCP not just click the +
